image_services/rtc_services/hkh/hkh_rgb.json

```diff
@@ -0,0 +1,21 @@
+{
+  "project_name": "HKH",
+  "s3_prefix": "HKH/",
```

Comment:
the prefix for this data set is `HKHwatermaps/` (i.e. the entire path is `s3://hyp3-nasa-disasters/HKHwatermaps/`)